jbmc/regression/jbmc/deterministic_assignments_json/clinit-state.json

```diff
@@ -0,0 +1,424 @@
+      "@type":"java.lang.Double",
+      "value":73.23
+    },
+    "genericInterfaceLess":{
+      "@type":"GenericImplementationLess",
+      "elements":[
+        {
+          "i":1,
+          "b":{
+            "x":5
+          }
+        }
+      ]
+    },
+    "referenceToOther":{
+      "@id":4,
+      "@type":"Foo",
+      "i":3,
+      "b":{
+        "x":4
+      }
+    },
+    "abstractClassFirst":{
+      "@type":"MyAbstractImplementationA"
+    },
+    "chineseCharField":{
+      "@type":"java.lang.Character",
+      "value":"字"
+    },
+    "booleanField":{
+      "@type":"java.lang.Boolean",
+      "value":true
+    },
+    "foo":{"@ref":4},
+    "genericInterfaceMore":{
+      "@type":"GenericImplementationMore",
+      "elementsT":{
+        "@type":"[LFoo;",
+        "@items":[
+          {
+            "i":1,
+            "b":{
+              "x":5
+            }
+          }
+        ]
+      },
+      "elementsU":{
+        "@type":"[LBar;",
+        "@items":[
+          {
+            "x":15
+          },
+          {
+            "x":12
+          }
+        ]
+      }
+    },
+    "byteField":{
+      "@type":"java.lang.Byte",
+      "value":98
+    },
+    "genericInterfaceSame":{
+      "@type":"GenericImplementation",
+      "elements":{
+        "@type":"[LFoo;",
+        "@items":[
+          {
+            "i":4,
+            "b":{
+              "x":6
+            }
+          },
+          {
+            "i":12,
+            "b":{
+              "x":14
+            }
+          }
+        ]
+      }
+    },
+    "asciiCharField":{
+      "@type":"java.lang.Character",
+      "value":"a"
+    },
+    "stringContainerField":{
+      "@type":"StringContainer",
+      "contained":"abcabcabcabcabc"
+    },
+    "referenceArrayField":{
+      "@id":3,
+      "@type":"[LFoo;",
+      "@items":[
+        {
+          "i":5,
+          "b":{
+            "x":-1
+          }
+        },
+        {
+          "i":6,
+          "b":{
+            "x":7
+          }
+        }
+      ]
+    },
+    "intField":{
+      "@type":"java.lang.Integer",
+      "value":610
+    },
+    "stringBufferField":{
+      "@type":"java.lang.StringBuffer",
+      "value":"bbbbbb"
+    },
+    "genericField":{
+      "@type":"GenericType",
+      "field":{
+        "@type":"Foo",
+        "i":3,
+        "b":{
+          "x":4
+        }
+      }
+    },
+    "concreteClassFirst":{
+      "@type":"MySubclassA"
+    },
+    "referenceToPrimitiveArray":{
+      "@id":2,
+      "@type":"[I",
+      "@items":[
+        5,55,610
+      ]
+    },
+    "interfaceSecond":{
+      "@type":"MyImplementationB",
+      "field":22
+    },
+    "primitiveArrayField":{"@ref":2},
+    "charArrayField":{
+      "@type":"[C",
+      "@items":[
+        "\u0000\u0001AB?;⿃"
+      ]
+    },
+    "fibCharField":{
+      "@type":"java.lang.Character",
+      "value":"ɢ"
+    },
+    "longField":{
+      "@type":"java.lang.Long",
+      "value":"202630"
```

Comment:
:no_entry_sign: Please make this long field bigger than the max int value to test how you are using std::stoi on previous commit

Reply:
Now equal to the max long value (`9_223_372_036_854_775_807L`).